.github/workflows/linters.yml

```diff
@@ -0,0 +1,56 @@
+name: Linters
+
```

Comment:
- [ ] Please Pull Request should have a descriptive title telling you what changes are inside.
- [ ] Please Pull Request should have a summary describing introduced changes in more detail.
![image](https://user-images.githubusercontent.com/74744787/176305291-3a8e9e1e-b9eb-4d6c-9db8-02ece206c116.png)